Dockerfile

```
@@ -3,8 +3,10 @@ RUN apt-get update && apt-get -y upgrade && apt-get install -y cmake libclang-de
 COPY . lighthouse
 ARG FEATURES
 ARG PROFILE=release
+ARG CARGO_USE_GIT_CLI=false
```

I'm feeling bullish, let's turn this on by default. The `Dockerfile` isn't used by our release builders anyway, so the risk of accidental slowdown is nil.

```suggestion
ARG CARGO_USE_GIT_CLI=true
```